students/johnn/mailroom/mailroom/ui.py

```diff
@@ -1,47 +1,56 @@
 #!/usr/bin/env python3
 
+"""
+menu system for donor program
+"""
+
 import sys
 from mailroom import security
 from mailroom.donors import Donors, Donor, save_donor_file, load_donor_file
 
-#security.user = None
 
 def safe_input(prompt=">"):
     """ Generic input routine. """
     # return null if anything goes wrong
     try:
-        selection=input(prompt).strip()
+        selection = input(prompt).strip()
     except (KeyboardInterrupt, EOFError):
         # don't exit the program on ctrl-c, ctrl-d
-        selection=""
+        selection = ""
     return selection
 
 
-def print_lines(lines=2,dest=sys.stdout):
+def print_lines(lines=2, dest=sys.stdout):
     """ Print variable number of linefeeds for clarity. """
 
-    for i in range(lines):
-        print("",file=dest)
+    for _ in range(lines):
+        print("", file=dest)
+
 
 def user_login():
     """ stub for user login """
     security.user = input("Enter your username: ")
 
+
 def user_logout():
     """ stub for user logout """
     security.user = None
 
```

Comment:
I think we have already talked about how this isn't the best way to implement security. :)